editor/editor_help.cpp

```diff
@@ -1840,10 +1976,12 @@ void EditorHelp::_help_callback(const String &p_topic) {
 	}
 }
 
-static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control *p_owner_node) {
+static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control *p_owner_node, EditorHelpImageDownloader *p_img_downloader) {
```

Comment:
What's the purpose of `p_img_downloader` if in every call it just uses the already available `img_downloader`?